homeassistant/components/metoffice/data.py

```diff
@@ -65,14 +76,36 @@ async def async_update(self):
     def _update(self):
         """Get the latest data from DataPoint."""
         if self._site is None:
-            _LOGGER.error("No Met Office forecast site held, check logs for problems")
+            _LOGGER.error("No Met Office sites held, check logs for problems")
```

Comment:
Looks like you may also be to cover this line in the tests to fulfill the coverage requirement.

Reply:
Didn't you add a test for this in the previous iteration? I think it got lost when you amended the last commit.